superset-frontend/packages/superset-ui-core/src/number-format/README.md

```diff
@@ -68,7 +68,7 @@ There is also a formatter based on [pretty-ms](https://www.npmjs.com/package/pre
 used to format time durations:
 
 ```js
-import { createDurationFormatter, formatNumber, getNumberFormatterRegistry } from from '@superset-ui-number-format';
+import { createDurationFormatter, formatNumber, getNumberFormatterRegistry } from '@superset-ui-number-format';
```

Comment:
This is an important flag of the duplicate word item, because it's in user facing documentation, and the documentation is wrong (but it isn't checked because no compiler checks code samples in docs).